src/PowerShellEditorServices/Workspace/Workspace.cs

```diff
@@ -174,6 +180,20 @@ public ScriptFile GetFile(string filePath)
         /// <param name="scriptFile">The out parameter that will contain the ScriptFile object.</param>
         public bool TryGetFile(string filePath, out ScriptFile scriptFile)
         {
+            try
+            {
+                if (filePath.Contains(":/") // Quick heuristic to determine if we might have a URI
```

Comment:
I thought I had suggested a code change like this before but we eventually decided that the `GetFile()` call on line 199 would throw in this case and we'd return false.  That said, I'd rather test this in a way that doesn't rely on catching an exception.

Reply:
Sorry @rkeithhill, are you for this change?

Reply:
Yes.

Reply:
My worry is that there seems to be a test that exists already for a `git:/` schemed file. My change would just reject those (and anything else that isn't `file`, `untitled` or `inmemory`). The fact that we already have a test for this makes me feel like there is an expectation that we can understand and handle a URI like this -- does that seem right?

Reply:
Those tests for `git:/` and `gitlens-git:/` look like my tests.  I think the intent here was to have those uri schemes return true for IsInMemory which we used to signal that you can't do things like set breakpoints with an "in memory" document.  In retrospect, perhaps that is not the best approach RE the way we used IsInMemory but the basic idea was to determine whether or not we had a file path we could provide to `Set-PSBreakpoint`.

Reply:
In an ideal design, I could walk up to the ScriptFile/EditorDocument and query a property like `CanHaveBreakpoints` or something like that to determine whether or not an LSP text document could have a breakpoint set on it.  But I think this is good as a stop-gap.